<ui version="4.0" >
 <class>SMPTransientWidget</class>
 <widget class="QWidget" name="SMPTransientWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of iterations</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sliderIterations" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="iterations" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Threshold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QDoubleSpinBox" name="threshold" >
     <property name="decimals" >
      <number>3</number>
     </property>
     <property name="minimum" >
      <double>0.001000000000000</double>
     </property>
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>271</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QPushButton" name="computeButton" >
     <property name="text" >
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderIterations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>iterations</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel" >
     <x>408</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderIterations</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>412</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel" >
     <x>278</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
